packages/api/src/controllers/room.ts

```diff
@@ -190,7 +190,16 @@ app.post(
         ttl: 5 * 60,
       }
     );
-    at.addGrant({ roomJoin: true, room: req.params.roomId });
+    at.addGrant({
+      roomJoin: true,
+      room: req.params.roomId,
+      ...(req.body.roomAdmin !== undefined && {
+        roomAdmin: req.body.roomAdmin,
+      }),
+      ...(req.body.canPublish !== undefined && {
+        canPublish: req.body.canPublish,
+      }),
```

Comment:
@victorges do you know of a neater way of doing this?

Reply:
Not really :/
The only trick I know, which I'm not sure would work here, is that `JSON.serialize` does not serialize `undefined` fields (as there's no value for that in JSON). So if this eventually gets JSON-serialized, you could just set `roomAdmin: req.body.roomAdmin` and it will work out. If we don't really control that code and think it might change in the future, I guess it could be risky.

One nit I'd suggest is destructuring the `req.body` into local variables, so these expressions are a little simpler. Something like this on line 184
```ts
const { name, roomAdmin, canPublish} = req.body;
```
then use the shoter vars around here. But it'd still need the `...(!x ? {} : { x })` logic anyway

Reply:
Oh another idea! Just noticed these are simple booleans, which probably have a default on the LiveKit side. Assuming the default for both is `false`, we can do 
```suggestion
      roomAdmin: req.body.roomAdmin ?? false,
      canPublish: req.body.canPublish ?? false,
```